pom.xml

```diff
@@ -1,236 +1,235 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>plugin</artifactId>
-        <version>4.71</version>
-    </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jenkins-ci.plugins</groupId>
+    <artifactId>plugin</artifactId>
+    <version>5.28</version>
+  </parent>
 
-    <groupId>io.jenkins.plugins</groupId>
-    <artifactId>redhat-dependency-analytics</artifactId>
-    <version>${revision}.${changelist}</version>
-    <packaging>hpi</packaging>
-    <properties>
-        <revision>0.9.1</revision>
-        <changelist>999999-SNAPSHOT</changelist>
-        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-        <jenkins.version>2.387.3</jenkins.version>
-    </properties>
-    <name>Red Hat Dependency Analytics Plugin</name>
-    <url>https://github.com/jenkinsci/redhat-dependency-analytics-plugin/blob/master/README.md</url>
-    <description>Add the ability to scan your dependencies and get an insight on the vulnerabilities, compliance, maintainability, popularity and more.</description>
+  <groupId>io.jenkins.plugins</groupId>
+  <artifactId>redhat-dependency-analytics</artifactId>
+  <version>${revision}.${changelist}</version>
+  <packaging>hpi</packaging>
+  <name>Red Hat Dependency Analytics Plugin</name>
+  <description>Add the ability to scan your dependencies and get an insight on the vulnerabilities, compliance, maintainability, popularity and more.</description>
+  <url>https://github.com/jenkinsci/redhat-dependency-analytics-plugin/blob/master/README.md</url>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.401.x</artifactId>
-                <version>2163.v2d916d90c305</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-        </dependencies>
-    </dependencyManagement>
+  <licenses>
+    <license>
+      <name>Apache License 2.0</name>
+      <url>http://www.apache.org/licenses/</url>
+    </license>
+  </licenses>
+  <scm>
+    <connection>scm:git:ssh://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/${gitHubRepo}.git</developerConnection>
+    <tag>0.1.0</tag>
+    <url>https://github.com/${gitHubRepo}</url>
+  </scm>
+  <properties>
```

Comment:
Are all versions defined here used ? e.g the ant version ? 

and we can move some dependency versions to here as well.